files/zh-tw/web/http/status/422/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 422 Unprocessable Content
+slug: Web/HTTP/Status/422
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
+---
+
+{{HTTPSidebar}}
+
+超文字傳輸協定（HTTP）**`422 Unprocessable Content`** 回應狀態碼表示伺服器理解請求實體的內容類型，並且請求實體的語法是正確的，但無法處理其中包含的指示。
```

Comment:
https://zh.wikipedia.org/zh-tw/%E8%B6%85%E6%96%87%E6%9C%AC%E4%BC%A0%E8%BE%93%E5%8D%8F%E8%AE%AE

```suggestion
超文本傳輸協定（HTTP）**`422 Unprocessable Content`** 回應狀態碼表示伺服器理解請求實體的內容類型，並且請求實體的語法是正確的，但無法處理其中包含的指示。
```